dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/master/registry/MasterRegistry.java

```diff
@@ -137,4 +142,7 @@ private String getLocalAddress(){
 
     }
 
+    public ScheduledExecutorService getHeartBeatExecutor() {
+        return heartBeatExecutor;
+    }
```

Comment:
It’s a bit redundant here. If it’s just for testing, I don’t think it’s necessary.

Reply:
> It’s a bit redundant here. If it’s just for testing, I don’t think it’s necessary.

I agree with @CalvinKirs .